pyproject.toml

```diff
@@ -1,3 +1,40 @@
 [build-system]
-requires = ["setuptools", "wheel", "cython"]
-build-backend = "setuptools.build_meta"
\ No newline at end of file
+requires = [
+    "setuptools>=61.2",
```

Comment:
This is causing the CI failure since Python 3.6 does not support `setuptools > 60.10`.

Reply:
Hm... do you plan to keep support for Python 3.6?

Reply:
We should probably drop support for 3.6, but would want to do that simultaneously for all of snappy's various components.  I suggest changing to `setuptools>=59.6` unless you're using some newer feature.

Reply:
Support for metadata in pyproject.toml was added in setuptools 61 I think,
and the [tool.setuptools] section is out of beta only since 68.1.0.

There's no urgency here.